### PR TITLE
PEP 689: Rename to "Unstable C API tier"

### PR DESCRIPTION
**Pull request by encukou**
Also includes a [change proposed in the initial review](https://github.com/python/peps/pull/2559#discussion_r858895879).

### CODE REVIEW

**Comment by AA-Turner** [comment]
I've definitely signed the CLA... 

Adam

File: pep-0689.rst [hunk 1]
```diff
@@ -90,16 +94,16 @@ Changes during the Beta period
 Since the API itself can change continuously up until Beta 1 (feature freeze)
 of a minor version, major users of this API are unlikely to test
 Alpha releases and provide feedback.
-It is very difficult to determine what needs to be exposed as semi-stable.
+It is very difficult to determine what needs to be exposed as unstable.
 
-Additions to the semi-stable tier will count as *stabilization*,
+Additions to the unstable tier will count as *stabilization*,
 and will be allowed up to Release Candidate 1.
```
Comment (AA-Turner) [inline pep-0689.rst:100]: This sentence now reads oddly, although it makes sense on the second reading (I think!)

A

**Review by AA-Turner — APPROVED** [review]

File: pep-0689.rst [hunk 2]
```diff
@@ -29,9 +29,12 @@ The Python C-API is currently divided into `three tiers <https://devguide.python
   <387>`, and requires deprecation warnings before changes
 - Private (internal) API, which can change at any time.
 
-We need a tier between Public and Private to accommodate tools like
-advanced debuggers and JIT compilers, which need access to CPython
-internals but assume the C-API they use does not change in patch releases.
+Tools requring access to CPython internals (e.g. advanced
+debuggers and JIT compilers) are often built for minor series releases
+of CPython, and assume that the C-API interals used do not change
```
Comment (JelleZijlstra) [inline pep-0689.rst:34]:
```suggestion
of CPython, and assume that the C-API internals used do not change
```

File: pep-0689.rst (hunk 1 above)
Comment (encukou) [reply pep-0689.rst:100]: Yes, the terminology is not perfect -- there's something more unstable than the unstable API. 
But as discussed in the thread, it's better than “semi-stable”.

File: pep-0689.rst (hunk 2 above)
Comment (encukou) [reply pep-0689.rst]: Thanks!

**Comment by encukou** [comment]
> My email notification said I need to sign it, seems the bot updated the status already

Sorry for the noise. I tried to attribute the commit to you, but the bot didn't like it, so I force-pushed with myself as `Author`.
I mentioned you in the squashed commit message instead.

**Comment by CAM-Gerlach** [comment]
@encukou You could use `Co-authored-by:` instead for that, in the future?

**Comment by AA-Turner** [comment]
He did: https://github.com/python/peps/commit/5232fda0ab13254ce6904987d1f2373cd112c5fa

```
PEP 689: Rename to "Unstable C API tier" (#2576)

Co-authored-by: Adam Turner <9087854+aa-turner@users.noreply.github.com>
```

(I don't know what happened with the CLA bot though, or if it will happen again)

A

**Comment by CAM-Gerlach** [comment]
Perhaps it didn't like your GitHub autogenerated address?

**Pull request merged**